source/locale/zh_CN/LC_MESSAGES/tutorial/20_recipes/008_specify_params.po

```diff
@@ -0,0 +1,146 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2018, Optuna Contributors.
+# This file is distributed under the same license as the Optuna package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Optuna 2.6.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-03-20 13:35-0400\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.8.0\n"
+
+#: ../../source/tutorial/20_recipes/008_specify_params.rst:6
+msgid ""
+"Click :ref:`here "
+"<sphx_glr_download_tutorial_20_recipes_008_specify_params.py>`     to "
+"download the full example code"
+msgstr ""
+
+#: ../../source/tutorial/20_recipes/008_specify_params.rst:15
+msgid "Specify Hyperparameters Manually"
+msgstr "手动指定超参数"
+
+#: ../../source/tutorial/20_recipes/008_specify_params.rst:17
+msgid ""
+"It's natural that you have some specific sets of hyperparameters to try "
+"first such as initial learning rate values and the number of leaves. "
+"Also, it's also possible that you've already tried those sets before "
+"having Optuna find better sets of hyperparameters."
+msgstr ""
+"你自然有一些特定的超参数集要先尝试, 比如初始学习率和叶子数量. "
+"另外, 也有可能在让 Optuna 找到更好的超参数集之前，你已经尝试过一些集合."
+
+#: ../../source/tutorial/20_recipes/008_specify_params.rst:22
+msgid "Optuna provides two APIs to support such cases:"
+msgstr "Optuna 提供 两个API 以应对这种场景:"
+
+#: ../../source/tutorial/20_recipes/008_specify_params.rst:24
+msgid ""
+"Passing those sets of hyperparameters and let Optuna evaluate them - "
+":func:`~optuna.study.Study.enqueue_trial`"
+msgstr "将这些超参数集合传递过去并让 Optuna 对其求值 - "
+":func:`~optuna.study.Study.enqueue_trial`"
+
+#: ../../source/tutorial/20_recipes/008_specify_params.rst:25
+msgid ""
+"Adding the results of those sets as completed ``Trial``\\s - "
+":func:`~optuna.study.Study.add_trial`"
+msgstr "将这些集合的结果标记为已完成的 ``Trial``\\s - "
+":func:`~optuna.study.Study.add_trial`"
+
+#: ../../source/tutorial/20_recipes/008_specify_params.rst:28
+msgid "First Scenario: Have Optuna evaluate your hyperparameters"
+msgstr "第一个场景: 让 Optuna 对你的超参数求值"
+
+#: ../../source/tutorial/20_recipes/008_specify_params.rst:30
+msgid ""
+"In this scenario, let's assume you have some out-of-box sets of "
+"hyperparameters but have not evaluated them yet and decided to use Optuna"
+" to find better sets of hyperparameters."
+msgstr "在这个场景中, 我们假设你已经有了一些开箱即用的超参数但是还没有对他们"
+"进行过求值, 你决定用 Optuna 来找到更好的超参数."
```

Comment:
```suggestion
"进行过求值, 你决定用 Optuna 来找到更好的超参数."
```
Deleted that character. 🙇

Reply:
Ops, sorry didn't notice this but instead submitted another commit.